src/ambianic/pipeline/avsource/av_element.py

```diff
@@ -10,7 +10,8 @@
 import requests
 from ambianic.util import stacktrace
 from ambianic.pipeline import PipeElement
-from ambianic.pipeline.avsource import gst_process
+from ambianic.pipeline.avsource import gst_process, picam
+from ambianic.pipeline.avsource.picam import Picamera
```

Comment:
Would this Picamera import work on a non-raspberry pi machine? Do we need to include conditional logic to only enable and make use of picam libs if its running on rpi?

Reply:
`picam ` is a wrapper for the "real" picamera and will return an error flag if picamera import fails